plugins-bundled/internal/plugin-admin-app/src/pages/PluginDetails.tsx

```diff
@@ -11,13 +12,15 @@ import { usePlugin } from '../hooks/usePlugins';
 import { Page } from 'components/Page';
 import { Loader } from 'components/Loader';
 
-export const PluginDetails = ({ query }: AppRootProps) => {
-  const { slug } = query;
+export const PluginDetails = ({ onNavChanged }: AppRootProps) => {
+  // onNavChanged(undefined as any);
```

Comment:
@dprokop -- I'm suspicious that something changed in our app handling --- calling `onNavChanged` crashes :(